src/pybind/mgr/diskprediction_local/requirements.txt

```diff
@@ -1,3 +1,6 @@
-numpy==1.15.1
-scikit-learn==0.19.2
-scipy==1.1.0
\ No newline at end of file
+numpy==1.16.4
```

Comment:
Do you need to update the dependencies in the rpm spec file as well? We have a distribution package called `mgr-diskprediction-local`

Reply:
Yes, you're correct I might have to update those dependencies. However, the rgf-python package is not available to install as "dnf install python3-rgf-python". So is there any way to just install the dependencies from requirements.txt without changing the rpm spec?

Reply:
> However, the rgf-python package is not available to install as "dnf install python3-rgf-python". So is there any way to just install the dependencies from requirements.txt without changing the rpm spec?

Yeah, I don't think so.

Reply:
@chauhankaranraj do you really need those updated dependencies? I dont see any `rgf-python` references in your code.

Reply:
Yeah, I need rgf-python because the trained models are instances of the "RGFClassifier" defined in that library. joblib/pickle imports rgf-python _under the hood_ when the [load](https://github.com/chauhankaranraj/ceph/blob/master/src/pybind/mgr/diskprediction_local/predictor.py#L188) function is called - that is why you won't see a direct reference in the code.

Reply:
@chauhankaranraj if we want to use rgf-python in production environment. we need to get it packaged. one option is to package it as a separated sub-package in ceph. and let `ceph-mgr-diskprediction-local` depend on it.

Reply:
@tchaikov Could you please help with the packaging part? I haven't done it before and not sure how package dependencies are managed in ceph

Reply:
@chauhankaranraj to maintain a python package is not trivial thing. and more importantly, i have concerns regarding to how the downstream need to tackle with this. as strictly speaking, python-rgf and/or python3-rgf is a ceph package. how shall we maintain it in long term?

could you send a mail to dev@ceph.io and ceph-maintainers@ceph.io mailing lists for more inputs from the community? if we are good and have a concrete plan with this direction, i can help on this.

Reply:
@tchaikov you're right, I'll send the mail. In the meanwhile, I used a scikit-learn model for seagate drives (performance metrics [here](https://nbviewer.jupyter.org/github/chauhankaranraj/ceph_drive_failure/blob/master/multiclass_clf.ipynb#Vanilla-Random-Forest) that will not have dependency issues. Should we merge this and then once we have a better plan for how to use python libraries (such as tensorflow, pytorch, etc) that aren't available as rpm/deb, I can update the predictor model as well as add models for other manufacturers.

Reply:
> Should we merge this

@chauhankaranraj what do you mean by "this" ?

Reply:
@tchaikov Sorry for not being clear. By "this" I meant the model built using just scikit-learn (so that we would not have any dependency issues). I updated my branch to use this model, and also updated parts of code as you suggested. Could you please review?

Could you also help me in updating the rpm spec file (I need to add pandas, joblib, and scikit-learn packages)?

Weird side note: the requirements.txt inside diskprediction_local lists scikit-learn as a dependency, but the rpm spec does not have any mention of scikit-learn...so how is it working right now?